packages/angular_devkit/build_angular/src/utils/environment-options.ts

```diff
@@ -80,6 +80,13 @@ export const cachingBasePath = (() => {
   return cacheVariable;
 })();
 
+// Persistent build cache
+const persistentBuildCacheVariable = process.env['NG_PERSISTENT_BUILD_CACHE'];
```

Comment:
I am open to change the name of this env variable.

Reply:
I'm fine with that name.

I think this should interact with `NG_BUILD_CACHE` though which should probably override.  Maybe add it as the first check in `getCacheSettings`?

Reply:
It already interacts with `NG_BUILD_CACHE`

In the below check 
```
const persistentBuildCacheVariable = process.env['NG_PERSISTENT_BUILD_CACHE'];
export const persistentBuildCacheEnabled =
  !cachingDisabled &&
  isPresent(persistentBuildCacheVariable) &&
  isEnabled(persistentBuildCacheVariable);
```

`cachingDisabled` is the value of `NG_BUILD_CACHE`.

Reply:
https://github.com/angular/angular-cli/blob/573160352c83e0460b816a65ceb8ad94f7ce2f3f/packages/angular_devkit/build_angular/src/utils/environment-options.ts#L69-L88